install/infra/single-cluster/aws/Makefile

```diff
@@ -41,7 +45,11 @@ apply-cluster:
 	@terraform apply -target=module.eks --auto-approve
 
 .PHONY: apply-tools
-apply-tools: install-cm-edns install-cluster-issuer
+apply-tools: install-cm-edns install-cluster-issuer install-cluster-autoscaler
```

Comment:
@nandajavarma Can you take a look on why this fails? I followed the same pattern but still get the following error:
<img width="1130" alt="Screenshot 2022-09-02 at 9 44 27 AM" src="https://user-images.githubusercontent.com/7892868/188057661-f3318b3b-3b99-4aaf-b434-cc075b576cff.png">



Seems to be because it was using a stale kubeconfig